packages/design-system/src/components/Table/Table.jsx

```diff
@@ -128,22 +110,17 @@ Table.propTypes = {
   /**
    * Applies responsive styles to vertically stacked rows at different viewpoint sizes.
    * When `stackBreakpoint` is set, `id` prop is required in `TableHeaderCell` and
-   * `headers` prop is required in `TableDataCell`.
+   * `headers` prop is required in `TableDataCell` or `TableHeaderCell` for rows with a header column.
    */
   stackBreakpoint: PropTypes.oneOf(['sm', 'md', 'lg']),
   /**
    * A striped variation of the table.
    */
   striped: PropTypes.bool,
   /**
-   * Applies a horizontal scrollbar and scrollable notice when the `Table`'s contents exceed the container width.
+   * Applies a horizontal scrollbar and scrollable notice on `TableCaption` when the `Table`'s contents exceed the container width.
    */
   scrollable: PropTypes.bool,
-  /**
-   * Additional text or content to display when the horizontal scrollbar is visible to give the user notice of the scroll behavior.
-   * This prop will only be used when the `Table` `scrollable` prop is set and the table width is wider than the viewport.
-   */
-  scrollableNotice: PropTypes.node,
```

Comment:
I still like the `scrollableNotice` to be inside the parent `Table` component, this way its right next to the `scrollable` prop. Is it possible to pass the `scrollableNotice` to the `TableCaption` children? Maybe through the `TableContext`?

Reply:
Keeping it consistent including it with the extending TableCaption props